### PR TITLE
Update packages

### PR DESCRIPTION
**Pull request by EnomaDebby**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,7 +30,9 @@
     "dotenv": "^4.0.0",
     "json-loader": "latest",
     "jsonfile": "^3.0.1",
+    "mem": "^4.0.0",
     "mysql": "^2.14.1",
-    "req": "^0.1.4"
+    "req": "^0.1.4",
+    "stringstream": "0.0.6"
```
Comment (pjrebsch) [inline package.json]: I'm not too familiar with yarn, but I still don't think that this is necessary. If this dependency is removed, the `0.0.6` version upgrade in `yarn.lock` still persists for `stringstream@~0.0.4` after running `yarn install`.

File: package.json [hunk 2]
```diff
@@ -30,7 +30,9 @@
     "dotenv": "^4.0.0",
     "json-loader": "latest",
     "jsonfile": "^3.0.1",
+    "mem": "^4.0.0",
```
Comment (pjrebsch) [inline package.json]: Also, why was this added?

File: package.json (hunk 1 above)
Comment (EnomaDebby) [reply package.json]: I can't update the breaking packages without doing this. I used yarn upgrade and this was added

File: package.json (hunk 2 above)
Comment (EnomaDebby) [reply package.json]: It's also a package with a vulnerability

Comment (pjrebsch) [reply package.json]: Ah sorry, it's a new one, I missed that. It seems that the original `1.1.0` version is still hanging around:

https://github.com/execonline-inc/vimeo-upload-lambda/pull/8/files#diff-8ee2343978836a779dc9f8d6b794c3b2R25998

This could be trickier to fix since it isn't just a patch version difference.

Looks like its being used be `os-locale`.

Comment (EnomaDebby) [reply package.json]: I rebuilt yarn.lock.

Comment (pjrebsch) [reply package.json]: Okay, now it looks nothing is using `mem` anymore in `yarn.lock`, so I think it can be removed as a dependency.

File: package.json (hunk 1 above)
Comment (pjrebsch) [inline package.json]: Same here—I don't see `stringstream` used by anything anymore in `yarn.lock`.

File: package.json (hunk 2 above)
Comment (EnomaDebby) [reply package.json]: Updated @pjrebsch

**Review by pjrebsch — APPROVED** [review]
Looks good to me 👍
